packages/nextjs-contentful-renderer/src/generateMdx/generateMdx.ts

```diff
@@ -20,7 +20,9 @@ export const generateMdx = async (blocks: Block[]): Promise<Model[]> => {
   });
 
   const all = await Promise.all(promises);
-  return all.reduce((acc, val) => acc.concat(val), []);
+  // .filter added to remove empty models, (ex. AiTextGenerator, but could be more in the future)
+  // prevents site rendering from breaking when a model is empty
+  return all.reduce((acc, val) => acc.concat(val), []).filter((model) => !!Object.keys(model).length);
```

Comment:
I had to put this in because it was breaking when `AiTextGenerator` instances were returned with `''`. So, I would assume if other scenarios could result in an empty `model`, those would break as well and this would fix it. The whole `model` would have to be an empty string, for this to filter it, so I would think this is safe.